recipes/enterprise-pulsar/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "enterprise-pulsar" %}
+{% set version = "3.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/enterprise-pulsar-{{ version }}.tar.gz
+  sha256: 598fe013878163c006bc90f4f4a91358dede78f270b764dc6b356a2d5d1e1d65
+
+build:
+  number: 0
+  skip: true  # [py27]
+  skip: true  # [win]
```

Comment:
Can this recipe be `noarch: python`?  

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

Reply:
I'm not sure. I know some of the dependencies do not support Windows. I really have no plans to support Windows but I could try and see if the CI passes...

Reply:
It doesn't have to support windows at all. Noarch python is only built on linux. If a dep is not available on a platform, the package simply won't install.

Reply:
> Note noarch is a statement about the package’s source code and not its install environment. A package is still considered noarch even if one of its dependencies is not available on a given platform. If this is the case, conda will display a helpful error message describing which dependency couldn’t be found when it tries to install the package. If the dependency is later made available, your package will be installable on that platform without having to make any changes to the feedstock. However, keep in mind that since noarch packages are built on Linux, all dependencies must be available on Linux.

^ from the docs which explains it better

Reply:
OK. I'm trying it now with a python >=3.6 constraint.